<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="9" passed="3" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-07-19T12:49:57 IST" name="Default suite" finished-at="2020-07-19T12:49:58 IST" duration-ms="1358">
    <groups>
    </groups>
    <test started-at="2020-07-19T12:49:57 IST" name="Default test" finished-at="2020-07-19T12:49:58 IST" duration-ms="1358">
      <class name="org.weatherComparision.testcases.WeatherDataComparisionTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:57 IST" name="setup" finished-at="2020-07-19T12:49:57 IST" duration-ms="44" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="compareLiveTemps(java.lang.String)[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:57 IST" name="compareLiveTemps" data-provider="getCities" finished-at="2020-07-19T12:49:57 IST" duration-ms="277" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Lucknow]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at org.weatherComparision.testcases.WeatherDataComparisionTest.compareLiveTemps(WeatherDataComparisionTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- compareLiveTemps -->
        <test-method is-config="true" signature="setup()[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:57 IST" name="setup" finished-at="2020-07-19T12:49:58 IST" duration-ms="60" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="compareLiveTemps(java.lang.String)[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="compareLiveTemps" data-provider="getCities" finished-at="2020-07-19T12:49:58 IST" duration-ms="61" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Bhopal]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- compareLiveTemps -->
        <test-method is-config="true" signature="setup()[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="setup" finished-at="2020-07-19T12:49:58 IST" duration-ms="29" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="compareLiveTemps(java.lang.String)[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="compareLiveTemps" data-provider="getCities" finished-at="2020-07-19T12:49:58 IST" duration-ms="48" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ajmer]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at org.weatherComparision.testcases.WeatherDataComparisionTest.compareLiveTemps(WeatherDataComparisionTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- compareLiveTemps -->
        <test-method is-config="true" signature="setup()[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="setup" finished-at="2020-07-19T12:49:58 IST" duration-ms="32" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="compareLiveTemps(java.lang.String)[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="compareLiveTemps" data-provider="getCities" finished-at="2020-07-19T12:49:58 IST" duration-ms="41" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Coimbatore]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at org.weatherComparision.testcases.WeatherDataComparisionTest.compareLiveTemps(WeatherDataComparisionTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- compareLiveTemps -->
        <test-method is-config="true" signature="setup()[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="setup" finished-at="2020-07-19T12:49:58 IST" duration-ms="28" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="compareLiveTemps(java.lang.String)[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="compareLiveTemps" data-provider="getCities" finished-at="2020-07-19T12:49:58 IST" duration-ms="41" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Mumbai]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- compareLiveTemps -->
        <test-method is-config="true" signature="setup()[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="setup" finished-at="2020-07-19T12:49:58 IST" duration-ms="24" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="compareLiveTemps(java.lang.String)[pri:0, instance:org.weatherComparision.testcases.WeatherDataComparisionTest@36c88a32]" started-at="2020-07-19T12:49:58 IST" name="compareLiveTemps" data-provider="getCities" finished-at="2020-07-19T12:49:58 IST" duration-ms="37" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Kolkata]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- compareLiveTemps -->
      </class> <!-- org.weatherComparision.testcases.WeatherDataComparisionTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
